src/Perl6/Actions.nqp

```diff
@@ -1427,11 +1427,31 @@ class Perl6::Actions is HLL::Actions does STDActions {
         my $doc := %*COMPILING<%?OPTIONS><doc>;
         if $doc {
             my $block := $*W.push_lexpad($/);
-
-            my $renderer := "Pod::To::$doc";
-
-            my $module := $*W.load_module($/, $renderer, {}, $block);
-
+            my $rx := /
+                       ^
+                       $<name> = ( [ \w | '::' ] + )
+                       [ ':' $<part> = ( [ 'ver' | 'auth' | 'api' ] )
```

Comment:
This would match "::a::::bbbbb". I think rakudo has a canonical parser for this, can you please look this up?
Also :ver<0.1>:ver<33>. You might end up with something you can't instantiate and fail later.

Reply:
a) Don;t think so because : must be followed by ver/auth/api. : (::bbbbb) and :: (:bbbbb) would fail.
b) I am certain it does, but where? and how could I look this up? nqp documentation is not so good. I have looked at everything on nqp site.